PocketKit/Sources/PocketKit/Item/ReadableHostViewController.swift

```diff
@@ -121,8 +121,12 @@ class ReadableHostViewController: UIViewController {
     @objc
     private func moveFromArchiveToSaves() {
         readableViewModel.moveFromArchiveToSaves { [weak self] success in
-            if success, let items = navigationItem.rightBarButtonItems, let index = items.firstIndex(of: getMoveFromArchiveToSavesButton) {
-                self?.navigationItem.rightBarButtonItems?[index] = getArchiveButton
+            if success,
+               let items = self?.navigationItem.rightBarButtonItems,
+               let getMoveFromArchiveToSavesButton = self?.getMoveFromArchiveToSavesButton,
+               let index = items.firstIndex(of: getMoveFromArchiveToSavesButton),
+               let archiveButton = self?.getArchiveButton {
+                self?.navigationItem.rightBarButtonItems?[index] = archiveButton
```

Comment:
due to 
![image](https://user-images.githubusercontent.com/6743397/228951200-f22f65d4-a5d6-4ef7-b9bc-e9000073daa9.png)

Reply:
In future I'd avoid adding unrelated changes to a PR.
It can complicate tracking down where issues/changes came from.